src/main/java/net/smartcosmos/dto/relationships/RelationshipUpdateMoniker.java

```diff
@@ -0,0 +1,29 @@
+package net.smartcosmos.dto.relationships;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Setter;
+
+import java.beans.ConstructorProperties;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+public class RelationshipUpdateMoniker {
```

Comment:
For Relationship, Update actually has no extra DTO, because update and create use the same endpoint with the same request body, therefore `RelationshipUpdate` should also contain all those fields. It doesn't matter that most of them would get ignored.

That `moniker` is the only one that effectively gets updated is something totally different and could differ with other implementations of the REST or persistence layers.